egg.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Page</title>
+    <link rel="stylesheet" href="index.css">
+
+</head>
+
+<body style = background:rgb(27,38,59)>
+    
+
+    <!---creating header-->
+    <header class="header">
+        <p id="myheader">Ed's Kitchen </p>
+        
+         <nav>
+                <a class="nav_item" href="index.html"> Home</a>
+                <a class="nav_item" href =about-me.html >About me </a>
+         </nav>
+    </header>
+
+
+     <!--will contain a card
+    Each card will contain a title, an image, an unordered list of indgrients, and a list of instructions
+    instructions contain a div with a two nested divs for the title and steps-->
+    <main>
+        <div class="card">
+            <h1 class="title2">Omelette</h1>
+            <div class="card-body">
+                <img class="card-img" src="http://www.nishkitchen.com/wp-content/uploads/2012/09/Spinach-Omelette-2B.jpg"> 
```

Comment:
Make sure to add an alt="" text to your images so that people still know what's happening on the page when things don't load correctly or a link breaks!